home-work.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+	"unicode/utf8"
+)
+
+func average(arr []int) float64 { // Task 1
+	if len(arr) == 0 {
+		return 0
+	}
+	sum := 0
+	for _, a := range arr {
+		sum += a
+	}
+	return float64(sum) / float64(len(arr))
+}
+
+func max(words []string) string { // Task 2.1
+	word := ""
+	for _, a := range words {
+		if utf8.RuneCountInString(a) > utf8.RuneCountInString(word) {
+			word = a
+		}
+	}
+	return word
+}
+
+func reverse(arr []int64) []int64 { // Task 2.2
+	newSlice := make([]int64, len(arr))
+	for i, a := range arr {
+		newSlice[i] = a
```

Comment:
You can use `copy` function to copy values from one slice to another.

Reply:
Thanks for the advice, next time i use `copy`